blog/_posts/2021-08-09-avida-ed-evolution.md

```diff
@@ -0,0 +1,528 @@
+4. I copied, pasted, modified my poc dragbar into the codebase, and tweaked a lot of things (it turned out) to make it actually compatible with the rest of the Avida-ED code and layout (Hello, maquetta grid!).
+
+This might be a great time to bring up..
+
+### The Importance of Proof of Concept
+   #### (Image of Proof of Concept)
+   What I learned is that, if you ever run into a situation where you don't know what the heck is going on in the codebase, and you don't know where to start? Do a proof of concept, it will save you big time. First, it is a great chance for you get yourself familiar with the Javascript functions that will be handy for your task. Second, it is a medium to communicate to stakeholders or your mentors what you are working on and how you plan on implementing the component onto the actual website. And lastly, and perhaps most importantly, it gives you a template, a kind of jumping off point to get you going (and with confidence) when you actually touch the codebase. Fear not! 
+
+When you do actually start working on the code though..
+
+### Approach code like a surgeon 
+What I mean by that is, my honest impression of working on the codebase was me constantly feeling like I was making small incisions in the code to insert whatever solution I had to offer the website (in this case, a dragbar). Like all beginner programmers, I was fearful. What if I break something really bad, and I mess something up, and the world comes to an end? (right...) Anyhow, I was always careful what I tweak and how I tweak it. I am a forgetful person, too. So I left a bunch of "yemi:(comment)" comments where I was changing anything, first leaving some breadcrumbs for me to pick back up if I do end up breaking something, and second to communicate to Diane what I was changing and why I was changing it, so if I do break something, she can at least follow what I was doing to perhaps offer up some higher level insight..(thank you, Diane!) But I learned that, if you approach code like a surgeon, and be very aware of every change you make precisely, you will save much energy later down the road where you need to back track not knowing where you messed up if you do (and you most likely will!) 
+
+### (Possibly essential code snippets)
+
+Still working on it!
+
+Dragula, the New Way to Drag and Drop (perhaps a more clever title)
+-- 
+It was a web of dependencies from which I knew no escape. And then, I saw the light. Its name was 'Dragula: Drag and Drop So Easy It Hurts," and yes I was hurt. It was thanks to Raheem, a participant who worked on the Avida-ED project before me, who did his dilligent research looking for the best drag and drop library to support the next generation of Avida-ED. His very neat proof of concept website was very helpful(see? it does help!). Turns out, drag and drop forms the foundation of all actions on Avida-ED. I was ambitious, and Diane was patient!
+
+# The Deets
```

Comment:
perhaps also include some before and after pics? this seems like a great improvement!